showtimes_session/src/lib.rs

```diff
@@ -170,7 +171,7 @@ impl ShowtimesRefreshClaims {
     fn new(user: showtimes_shared::ulid::Ulid) -> Self {
         let iat = jiff::Timestamp::now();
         // Refresh claims last for 90 days
-        let exp = iat + jiff::SignedDuration::new(7 * 24 * 60 * 60, 0);
+        let exp = iat.checked_add(90.days()).unwrap_or(jiff::Timestamp::MAX);
```

Comment:
This will always fail. See: https://docs.rs/jiff/latest/jiff/struct.Timestamp.html#errors-7